1-sum-app/sum.html

```diff
@@ -0,0 +1,15 @@
+<html>
+    <head>
+        <title>1-sum-app</title>
+    </head>
+    <body>
+        <div class="wrapper-content">
+            <u><h1>Add Two Numbers </h1></u><br>
+            <div>
+                <label>First number: </label><input type="number"><br><br>
```

Comment:
Please avoid using `<br>` in html. If we need to move the element to next line, it's better to wrap it in `<div> </div>`. Also, can add margin-top or margin-bottom if more space is needed.